Makefile

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/make
+
+NAME = "ProductOpener"
+DEV_ARGS = -f docker-compose.yml -f docker/dev.yml -f docker/mongodb.yml
+PROD_ARGS = -f docker-compose.yml -f docker/prod.yml -f docker/geolite2.yml
+
+#-----#
+# Dev #
+#-----#
+dev: up build_npm import_sample_data
+
+up:
+	docker-compose ${DEV_ARGS} up -d --remove-orphans --build backend frontend
+
+down:
+	docker-compose ${DEV_ARGS} down
+
+restart:
+	docker-compose ${DEV_ARGS} restart
+
+log:
+	docker-compose ${DEV_ARGS} logs -f
+
+import_sample_data:
+	docker-compose ${DEV_ARGS} exec backend bash /opt/product-opener/scripts/import_sample_data.sh
+
+status:
+	docker ps
+
+prune:
+	docker system prune -af
+
+#------------#
+# Production #
+#------------#
+prod: clean
+	docker-compose ${PROD_ARGS} up -d --remove-orphans
+
+#-------#
+# Build #
+#-------#
+build_npm:
+	docker run --rm -it -v $(PWD)/node_modules:/mnt/node_modules -v $(PWD):/mnt -w /mnt node:lts npm install
+	docker run --rm -it -v $(PWD)/node_modules:/mnt/node_modules -v $(PWD):/mnt -w /mnt node:lts npm run build
```

Comment:
`$(PWD)` doesnt't resolve to anything on my machine:
```
█Johannes@DESKTOP-TJK3GRM█  deploy-dcker██ restructure-docker ≡██100   16.7.0█ ﲵ pwsh█ █ make build_npm
docker run --rm -it -v /node_modules:/mnt/node_modules -v :/mnt -w /mnt node:lts npm install
npm WARN saveError ENOENT: no such file or directory, open '/mnt/package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN enoent ENOENT: no such file or directory, open '/mnt/package.json'
npm WARN mnt No description
npm WARN mnt No repository field.
npm WARN mnt No README data
npm WARN mnt No license field.

up to date in 0.42s
found 0 vulnerabilities

docker run --rm -it -v /node_modules:/mnt/node_modules -v :/mnt -w /mnt node:lts npm run build
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /mnt/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/mnt/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-08-24T16_13_46_821Z-debug.log
make: *** [build_npm] Fehler 254
```
I installed `GnuWin32.Make` with `winget` to run this. Do I need a different version of `make` for that variable?

With `$(realpath ./)` instead of `$(PWD)`, it worked without any issues: https://github.com/openfoodfacts/openfoodfacts-server/blob/15c78e4cdde11250e131956d0b359688510c671e/Makefile#L38-L40